src/js/snapshots/imageSnapshot.snapshot.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-undef */
+
+// page object is from jest-puppeteer API
+
+const config = {
```

Comment:
You can configure toMatchImageSnapshot before tests are happening in setup file, that way you do not need to keep that in test file nor add that to every method call

Reply:
fixed